### ISSUE TITLE
[Bug?]: Clerk Custom Flow Auth Strategy | Exception in getAuthenticationContext: Cannot find module './dist/instance' 

### ISSUE THREAD

**Issue by wispyco** [redwoodjs/redwood#6807]
### What's not working?

The following `currentUser` is not showing anything after a user has signed up with Clerk authentication strategy.

I pull it from the `@redwoodjs/auth`

```
  import { useAuth } from '@redwoodjs/auth'
  const { currentUser, isAuthenticated } = useAuth()
```

I am using clerks `useSignUp` hook to do most of the work for signing up.

The exact file exists here

https://github.com/wispyco/manage_members/blob/master/web/src/pages/SignupPage/SignupPage.tsx

I get the following error in terminal

```
🚨 graphql-server Error building context. Error: Exception in getAuthenticationContext: Cannot find module './dist/instance'
```

I followed this flow from clerk

https://clerk.dev/docs/authentication/custom-flows/password

The reason I did it this way was so I can be as custom as possible.

It does mention something about using `setSession(...)` but I don't know what I would pass inside. Although the session does seem to be set because if I try to signUp again I get an error saying session is already set.

Any help would be greatly appreciated. 

I am going to check the box for being interested in helping, but not sure I am fully capable to do so, but would try for sure.

### How do we reproduce the bug?

You can pull down this repo

https://github.com/wispyco/manage_members

Everything I am working on exists on the `/signup` page

All you do is fill in an email, and password. Receive the email code and enter it into the verification box and hit verify. 
In the console you will notice that the task completes meaning the user get's created.

After that I expected the `currentUser` to display.

Maybe the `api/src/lib/auth.ts` needs to be modified.

Thanks ahead of time.

### What's your environment? (If it applies)

```shell
System:
    OS: macOS 11.2.3
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.18.1 - /private/var/folders/wk/dzm9fmpn1hs1czk3hmd4vlrr0000gn/T/xfs-67f0b245/node
    Yarn: 3.2.3 - /private/var/folders/wk/dzm9fmpn1hs1czk3hmd4vlrr0000gn/T/xfs-67f0b245/yarn
  Databases:
    SQLite: 3.32.3 - /usr/bin/sqlite3
  Browsers:
    Chrome: 107.0.5304.87
    Firefox: 106.0.1
    Safari: 14.0.3
  npmPackages:
    @redwoodjs/core: 3.3.0 => 3.3.0
```

### Are you interested in working on this?

- [X] I'm interested in working on this

**Comment by jtoar** [comment]
Hey @wispyco, we think we've resolved this in https://github.com/redwoodjs/redwood/pull/6685. We'll get a patch out soon with that fix and report back when it's out and you can let us know if that solved it for you. Sound good?

**Comment by wispyco** [comment]
@jtoar Sounds great to me, thanks for checking in.

**Comment by jtoar** [comment]
This was released in v3.3.2 https://github.com/redwoodjs/redwood/releases/tag/v3.3.2
